apps/frontend/src/modules/projects/slices/project.slice.ts

```diff
@@ -41,17 +44,24 @@ const { actions, name, reducer } = createSlice({
 			state.project = null;
 			state.projectStatus = DataStatus.REJECTED;
 		});
+
 		builder.addCase(loadAll.pending, (state) => {
 			state.dataStatus = DataStatus.PENDING;
 		});
 		builder.addCase(loadAll.fulfilled, (state, action) => {
-			state.projects = action.payload.items;
+			const { items, totalItems } = action.payload;
+			const { page } = action.meta.arg;
+
+			state.projects =
+				page === EMPTY_LENGTH ? items : [...state.projects, ...items];
```

Comment:
I think you wanted to use your `const FIRST_PAGE = 0` instead. This is more correct semantically
```suggestion
				page === FIRST_PAGE ? items : [...state.projects, ...items];
```